cartographer/mapping/internal/3d/local_trajectory_builder_options_3d.cc

```diff
@@ -57,8 +58,10 @@ proto::LocalTrajectoryBuilderOptions3D CreateLocalTrajectoryBuilderOptions3D(
           parameter_dictionary->GetDictionary("ceres_scan_matcher").get());
   *options.mutable_motion_filter_options() = CreateMotionFilterOptions(
       parameter_dictionary->GetDictionary("motion_filter").get());
+  *options.mutable_pose_extrapolator_options() = CreatePoseExtrapolatorOptions(
+     parameter_dictionary->GetDictionary("pose_extrapolator").get());
```

Comment:
Did you run clang-format? I think this needs to be indented by 4.

Reply:
Good catch! Done.